.vscode/settings.json

```diff
@@ -25,6 +25,7 @@
         "--config=${workspaceFolder}/.flake8",
     ],
     "python.linting.mypyEnabled": true,
+    "python.linting.mypyPath": "/anaconda/envs/himl/bin/mypy",
```

Comment:
I added this here because `mypy` had been refusing to work with the `HimlHisto` environment. Is it reasonable to expect anyone wanting to use `mypy` linting to have the `himl` environment installed?